### DISCUSSION TITLE
Automation not working when using AG AU plugin, in plugins after the first one

### DISCUSSION THREAD

**Discussion by marioc-tech** [apohl79/audiogridder#1278] [Help] [answered]
Hello,

I would like to report an issue with the AU plugin of Audiogridder. 

When attempting to automate parameters on slots/plugins beyond the first one, the parameters can be assigned for automation, but when I attempt to adjust the parameter from the hosted plugin, it sends a zero value back to the DAW and stays at zero.
As a result, no parameters can be automated in plugins placed after the first plugin.
I have tested the issue multiple times and it is consistent.

Here is a video that shows the issue: https://drive.google.com/file/d/1EH4kXY3VrigJcd2AvqYomi8u6sSk_Jdf/view?usp=sharing

The issue occurs regardless of the hosted plugin and it happens only when using Audiogridder AU. The AG VST3 does not have this issue.

Thanks in Advance!

**Comment by marioc-tech** [comment]
> Hello,
> 
> I would like to report an issue with the AU plugin of Audiogridder.
> 
> When attempting to automate parameters on slots/plugins beyond the first one, the parameters can be assigned for automation, but when I attempt to adjust the parameter from the hosted plugin, it sends a zero value back to the DAW and stays at zero. As a result, no parameters can be automated in plugins placed after the first plugin. I have tested the issue multiple times and it is consistent.
> 
> Here is a video that shows the issue: https://drive.google.com/file/d/1EH4kXY3VrigJcd2AvqYomi8u6sSk_Jdf/view?usp=sharing
> 
> The issue occurs regardless of the hosted plugin and it happens only when using Audiogridder AU. The AG VST3 does not have this issue.
> 
> Thanks in Advance!

Has anyone checked this?!

**Comment by apohl79** [comment, marked as answer]
Thx for this report and the video! I created a bug ticket for this, so it can be addressed.
